src/CHIPBindings.cc

```diff
@@ -1319,10 +1320,97 @@ hipError_t hipMemcpyPeerAsync(void *Dst, int DstDeviceId, const void *Src,
   CHIP_CATCH
 };
 
-hipError_t hipMemcpyParam2DAsync(const hip_Memcpy2D *PCopy,
-                                 hipStream_t Stream) {
+static inline hipError_t hipMemcpyAsync_internal(void *Dst, const void *Src,
```

Comment:
Should we use a naming convention closer to the LLVM style?
```suggestion
static inline hipError_t hipMemcpyAsyncInternal(void *Dst, const void *Src,
```